packages/common/src/core/schemas/shared/HubItemEntitySchema.ts

```diff
@@ -30,6 +30,19 @@ export const HubItemEntitySchema: IConfigurationSchema = {
     categories: ENTITY_CATEGORIES_SCHEMA,
     isDiscussable: ENTITY_IS_DISCUSSABLE_SCHEMA,
     _thumbnail: ENTITY_IMAGE_SCHEMA,
+    _followers: {
```

Comment:
Does `_followers` need to be on the base schema, or should it only exist on `SiteSchema`?

Reply:
That's a good callout - I put it here because although the concept of a followers group only exists for sites today, I think the idea is that any item-backed entity _could_ have a followers group in the future. That's also why I've added the `IWithFollowers` and `IWithFollowersBehavior` to the `IHubItemEntity` and `HubItemEntity` respectively rather than just to the `HubSite`.

With that said, if you think it would make more sense to keep this scoped to sites until followers are actually implemented for other entities, I can move it for now

Reply:
Thanks makes sense to leave it for potential future use cases.  Looks good!

Reply:
We will add followers for Initiative and Project - the only reason it's on now Site is b/c that was/is synonymous with "Initiative".